lightning/lightning_daemon.js

```diff
@@ -14,14 +18,23 @@ const {SSS_LND_TLS_CERT} = process.env;
   @returns
   <LND GRPC API Object>
 */
-module.exports = ({}) => {
-  try {
+module.exports = ({network}) => {
+  switch (network) {
+  case 'testnet':
+    return lightningDaemon({
+      cert: SSS_TESTNET_LND_TLS_CERT,
+      host: SSS_TESTNET_LND_GRPC_HOST,
+      macaroon: SSS_TESTNET_LND_MACAROON,
+    });
+    break;
+  case 'ltctestnet':
     return lightningDaemon({
-      cert: SSS_LND_TLS_CERT,
```

Comment:
Let's change this so that it just takes advantage of the naming convention and looks for credentials that match the network name + convention